### DISCUSSION TITLE
Can distributed deployment use non-erasure code mode?

### DISCUSSION THREAD

**Discussion by Beau-Chen** [minio/minio#13729] [Q&A] [answered]
Dear project author, I recently tried to deploy distributed Minio on k8s.

 I found that it will automatically enable EC mode, but I have done related work on redundancy and data recovery in the underlying data storage. 

Can I not enable EC mode during distributed deployment? 

I'm looking forward to your reply.

Thanks sincerely :)

**Comment by klauspost** [comment, marked as answer]
Erasure Coding is mandatory for distributed installs.

**Reply by Beau-Chen** [reply]
Thank you very much :)
